iscsi/iscsi.go

```diff
@@ -350,6 +355,68 @@ func Disconnect(tgtIqn string, portals []string) error {
 	return err
 }
 
+// DisconnectVolume removes a volume from a Linux host.
+func DisconnectVolume(c Connector) error {
+	// Steps to safely remove an iSCSI storage volume from a Linux host are as following:
+	// 1. Unmount the disk from a filesystem on the system.
+	// 2. Flush the multipath map for the disk we’re removing (if multipath is enabled).
+	// 3. Remove the physical disk entities that Linux maintains.
+	// 4. Take the storage volume (disk) offline on the storage subsystem.
+	// 5. Rescan the iSCSI sessions.
+	//
+	// DisconnectVolume focuses on step 2 and 3.
+	// Note: make sure the volume is already unmounted before calling this method.
+	if c.Multipath {
```

Comment:
This works, thanks for thinking of it!  I think it would be good to add some additional logging but other than that it's a good first add IMO

Reply:
Added some log.
btw, what do you think of the name "DisconnectVolume", is it acceptable?